index.js

```diff
@@ -0,0 +1,212 @@
+    </div>
+    </div>`;
+    existingCard0.appendChild(recentcard);
+  } else if (index < 4) {
+    const card = document.createElement('div');
+    card.classList.add('cards1');
+    card.innerHTML = `<div class="card-2 all-cards desk-card-${index}">
+  <div class="no-hover">
+  <h2 class="prof-art">${project.Title2} </h2>
+  <h2 class="desk-prof-art-dash">${project.Titled}</h2>
+  <p class="p22 desk-p22-d">${project.description}</p>
+  <ul class="skils desk-skils-d">
+  <li class="button desk-skil button-d">${project.technologies[0]}</li>
+  <li class="skil button desk-skil button-d">${project.technologies[1]}</li>
+  <li class="skil button desk-skil button-d">${project.technologies[2]}</li>
+  </ul>
+  </div>
+  <button class ="see-project-2 desk-see-project-1c" id="seeproject-2" onclick="popupp()">see project</button>
+  </div>
+  `;
+    existingCard.appendChild(card);
+  } else {
+    const card = document.createElement('div');
+    card.classList.add('cards1');
+    card.innerHTML = `<div class="card-2 all-cards desk-card-${index}">
+    <div class="no-hover">
+    <h2 class="prof-art">${project.Title2} </h2>
+    <h2 class="desk-prof-art-dash">${project.Titled}</h2>
+    <p class="p22 desk-p22-d">${project.description}</p>
+    <ul class="skils desk-skils-d">
+    <li class="button desk-skil button-d">${project.technologies[0]}</li>
+    <li class="skil button desk-skil button-d">${project.technologies[1]}l</i>
+    <li class="skil button desk-skil button-d">${project.technologies[2]}</li>
+    </ul>
+    </div>
+    <button class ="see-project-2 desk-see-project-1c" id="seeproject-2" onclick="popupp()">see project</button>
+    </div>
+    `;
+    existingCard2.appendChild(card);
+  }
+});
+
+let i = 0;
+ProjectsMobile.forEach((projec) => {
+  const popup = document.createElement('div');
+  popup.classList.add('modal');
+  popup.innerHTML = ` <div class="modal-content modal-contentD">
+      <div class="modalHead-close modalHead-closeD">
+      <h2 class = "modalHeader modalHeaderD" id="modalHeader">${projec.Title}${[i]}</h2>
+      <span class="close closeD">${projec.ClosePop}</span>
+      </div>
+      <div class = "desk0">
+      <ul id="modalTech" class = "modalTech">
+      <li>${projec.TecnologiesPop[0]}</li>
+      <li>${projec.TecnologiesPop[1]}</li>
+      <li>${projec.TecnologiesPop[2]}</li>
+      </ul>
+      <div class ="desk1">
+      <img class ="modal-img" src="${projec.ImagePop[0]}" alt="${projec.ImagePop[1]}">
+      <div class ="desk2">
+      <p class ="modal-pm">${projec.DescriptionPop} </p>
+      <div class = "live-source live-sourceD">
+      <a href="${projec.LinksPop[0][0]}">${projec.LinksPop[0][1]}<img src="${projec.LinksPop[0][2]}" alt="${projec.LinksPop[0][3]}"></a>
+  <a href="${projec.LinksPop[1][0]}">${projec.LinksPop[1][1]} <img src="${projec.LinksPop[1][2]}" alt="${projec.LinksPop[1][1]}"></a>
```

Comment:
- [ ] Great job so far, but you have not implement the changes requested by the previouse reviewer.According to your previouse reviewer Please kindly adjust the spacing margin/padding on top, right, bottom, and left in the [Mobile & Desktop] design to ensure that the popup window is positioned at the center of the page to match the template as much as possible. 💪

Your design
![Screenshot 2023-05-27 210254](https://github.com/solog0039/Portfolio-final/assets/92458236/ff039f35-ecd1-4c95-8c35-7c651895af08)

- [ ] your design  on mobile

![Screenshot 2023-05-27 205931](https://github.com/solog0039/Portfolio-final/assets/92458236/b7499b5e-285f-4e6c-ada5-2db6ada228d3)

- [ ] figma design on mobile
![Screenshot 2023-05-27 211543](https://github.com/solog0039/Portfolio-final/assets/92458236/2295c86b-560a-43b1-9fa9-3c40c64f9d57)


- [ ] figma design

![fig](https://github.com/solog0039/Portfolio-final/assets/92458236/adb39007-658d-4345-bdbe-70eb6aff144a)

Reply:
- [ ] As requested by the previous reviewer kindly fix your design to match the given desgin

Your design
![image](https://github.com/solog0039/Portfolio-final/assets/81932478/c343bebd-1f1b-45d9-addb-e7fde3387b90)

Given design
![image](https://github.com/solog0039/Portfolio-final/assets/81932478/4f46944b-c057-4713-9f7b-bd315fcd2849)

Your Mobile Version
![image](https://github.com/solog0039/Portfolio-final/assets/81932478/0d2ae22c-a63b-4ec2-9490-591a345a06ca)

Given Template
![image](https://github.com/solog0039/Portfolio-final/assets/81932478/e8ed61e3-2bf5-45b1-a247-12ce35458ac9)